src/TodoList.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+
+let id = 0
+
+const LIST_TITLES = [
+    'Begin Todo app',
+    'Checkout new github branch',
+    'Create list',
+    'Display list on page',
+    'Push changes to github',
+    'Go to bed'
+];
+
+const buildList = (titles) => {
+    const todoList = [];
+    titles.forEach((title) => {
+        todoList.push(
+            {
+                id: ++id,
+                title: title,
+            }
+        );
+    });
+    return todoList;
+};
```

Comment:
Nitpick (doesn't require changes): 
These lines of code work well as intended, BUT there's another array method that accomplishes the same result with less code. Can you figure out which method to use?

<details>
<summary>Spoiler</summary>

```js
const buildList = (titles) => {
    let id = 0;
    return titles.map((title) => ({ id: ++id, title }));
};
```
[Array.map](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map) returns a new shallow copy of the `titles` array -- it doesn't mutate the original array -- but creates objects containing `id` and `title` properties!
</details>

Reply:
Good stuff! I didn't think about using map() that way. I'll start using that. Definitely cleaner.